WORKSPACE

```diff
@@ -73,3 +73,7 @@ initialize_external_repositories()
 load("@com_github_grpc_grpc//bazel:grpc_extra_deps.bzl", "grpc_extra_deps")
 
 grpc_extra_deps()
+
```

Comment:
I suspect this is related to:
> It was necessary to add and register cpython repository because without it, bazel_rules_hdl would fail to install its pip dependencies

This is surprising that this is required here, since XLS also install pip deps from https://github.com/google/xls/blob/main/dependency_support/pip_requirements.txt

Reply:
It looks like we can remove this workaround.